app/templates/layout.html

```diff
@@ -78,5 +78,6 @@
 
     <script src="{{ basepath }}/components/jquery/jquery.js"></script>
     <script src="{{ basepath }}/components/bootstrap/bootstrap-built.js"></script>
+    <script src="{{ basepath }}/js/equalHeightColumns.js"></script>
```

Comment:
If it's fine to load it at the end, then you can just put it on the page itself. Otherwise you can add a new block thing like you wrote, or you can put this line in a conditional.